azure-iot-device/azure/iot/device/common/pipeline/pipeline_stages_mqtt.py

```diff
@@ -182,16 +182,16 @@ def _run_op(self, op):
             try:
                 self.transport.connect(password=password)
             except Exception as e:
-                logger.error("transport.connect raised error")
-                logger.error(traceback.format_exc())
+                logger.info("transport.connect raised error")
```

Comment:
>info [](start = 23, length = 4)

I feel like this could probably remain debug - The user has no insight into what transport.connect is. It's not particularly "informative" - plus when the error gets raised they'll get something that's actually useful at high level

Reply:
I'm following the convention that errors that get returned from external code get logged at info level.  Yes,m the user doesn't know what transport.connect is, but this is useful in understanding why the code is changing behavior.  There's also a certain amount of "expectedness" that I'm factoring in.   We don't expect `transport.connect` to raise an exception.  If it does, we need to "call it out" using `logger.info`

---
In reply to: [478833073](https://github.com/Azure/azure-iot-sdk-python/pull/649#discussion_r478833073) [](ancestors = 478833073)